godel_robots/abb/abb_file_suite/include/abb_file_suite/abb_motion_ftp_downloader.h

```diff
@@ -0,0 +1,34 @@
+#ifndef ABB_MOTION_FTP_DOWNLOADER_H
+#define ABB_MOTION_FTP_DOWNLOADER_H
+
+#include <ros/subscriber.h>
+#include <ros/service_server.h>
+#include <trajectory_msgs/JointTrajectory.h>
+#include "abb_file_suite/ExecuteProgram.h"
+
+namespace abb_file_suite
+{
+
+class AbbMotionFtpDownloader
```

Comment:
Class level documentation describing the general interface and use of this object